client/src/api/email.js

```diff
@@ -1,21 +1,10 @@
-import store from '..'
+import apiRequest from './lib/apiRequest'
 
 const emailApiClient = {
-  send (projectInfo) {
-    const options = {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify({
-        project: projectInfo,
-        user: {
-          email: store.getState().user.email
-        }
-      })
-    }
-
-    return fetch('/email', options)
+  send (apiOptions, project, email) {
+    // TODO: fix this by supplying server emailController the sendgrid api key
```

Comment:
sendgrid api key should only be accessible from the server, so it can't be passed from the client-side.

Reply:
right! whoops, this comment definitely should have been in the server emailController. I get errors when applying to projects because the `emailClient.setApiKey(process.env.SENDGRID_API_KEY)` isn't in the `.env` file